api/src/main/java/org/eclipse/microprofile/metrics/Metadata.java

```diff
@@ -139,9 +142,8 @@
      * <p/>
      */
     Metadata() {
-        String globalTagsFromEnv = System.getenv(GLOBAL_TAGS_VARIABLE);
-
-        addTags(globalTagsFromEnv);
+        Optional<String> globalTags = ConfigProvider.getConfig().getOptionalValue(GLOBAL_TAGS_VARIABLE, String.class);
```

Comment:
This changes the semantic of GLOBAL_TAGS_VARIABLE that can not be configured in many different places (`microprofile-config.properties`, System properties, etc.) in addition to environment variables.

The section in 
https://github.com/eclipse/microprofile-metrics/blob/master/spec/src/main/asciidoc/architecture.adoc#supplying-of-tag should be updated to reflect that.

fiy, since mp-metrics now depends on mp-config to configure the global tags, the name of the configuration properties could be updated to something more Java-friendly such as `mp-metrics-tags` (or `mp.metrics.tags`). microprofile-config will automatically convert this name to the `MP_METRIC_TAGS` environment variable (making it backwards compatible)

Reply:
Right, I added a small change to the spec document, it now says that using `mp.metrics.tags` system property will work as well...

I was thinking whether it would be possible, for complete compatibility, still accept only an env variable through the MP Config API, but that would 1) not make much sense, 2) I think it's not possible with MP Config in a portable way. 
But I doubt that anyone has a `MP_METRICS_TAGS` system property which is intentionally NOT meant for being picked up by Metrics, who would now face the problem that it suddenly gets picked up.